News.md

```diff
@@ -2,6 +2,11 @@
 
 # [unreleaseed]
 
+# v1.9.2 (05-12-2024)
+
+- Make `TreeSitter::TreeSitterError < Exception` instead of `StandardErorr`;
+  we don't want them to be handled by default `rescue`.
+
```

Comment:
This could arguably be a breaking change. I think we should at least bump to `v1.10.0` or even `v2.0.0`.

Reply:
I agree with that. Let's bump to `1.10.0`. I prefer bumping major when tree-sitter breaks some APIs, but I'm not religious about that. You prefer `2.0.0`?

Reply:
1.10.0 is good with me 👍